pandas/tests/tseries/test_timezones.py

```diff
@@ -1767,3 +1767,22 @@ def compare_local_to_utc(tz_didx, utc_didx):
                                   tslib.maybe_get_tz('Asia/Tokyo'))
         tm.assert_numpy_array_equal(result, np.array(
             [tslib.iNaT], dtype=np.int64))
+
+
```

Comment:
use ``assert ut == expected`` (we are moving away from inherity from ``tm.TestCase`` entirely.